test/ts-scenario/features/gateway.feature

```diff
@@ -18,38 +18,39 @@ Feature: Configure Fabric using CLI and submit/evaluate using a network Gateway
-		Then The gateway named mycouchgateway has a error type response containing "Error: Expected 5 parameters, but 2 have been supplied"
+		Then The gateway named mycouchgateway has a error type response containing Error: Expected 5 parameters, but 2 have been supplied
 		When I use the gateway named mycouchgateway to evaluate a transaction with args [noSuchEvauateTransaction,1001] for contract fabcar instantiated on channel gatewaychannel
-		Then The gateway named mycouchgateway has a error type response containing "Error: You've asked to invoke a function that does not exist: noSuchEvauateTransaction"
+		Then The gateway named mycouchgateway has a error type response containing Error: You've asked to invoke a function that does not exist: noSuchEvauateTransaction
 		When I use the gateway named mycouchgateway to evaluate a transaction with args [queryCar,because,I,said,so] for contract fabcar instantiated on channel gatewaychannel
-		Then The gateway named mycouchgateway has a error type response containing "Error: Expected 1 parameters, but 4 have been supplied"
+		Then The gateway named mycouchgateway has a error type response containing Error: Expected 1 parameters, but 4 have been supplied
 
 	Scenario: Using a Gateway to submit transactions I can use different event handler strategies
 		When I modify mycouchgateway to submit a transaction with args [createCar,1002,Ford,Mustang,Silver,Andy] for contract fabcar instantiated on channel gatewaychannel using handler option MSPID_SCOPE_ALLFORTX
-		Then The gateway named mycouchgateway has a event type response matching "{\"Org1\":1,\"Org2\":0}"
+		Then The gateway named mycouchgateway has a event type response containing "status":"VALID"
 		When I modify mycouchgateway to submit a transaction with args [createCar,1003,Ford,Mustang,Silver,Andy] for contract fabcar instantiated on channel gatewaychannel using handler option MSPID_SCOPE_ANYFORTX
-		Then The gateway named mycouchgateway has a event type response matching "{\"Org1\":1,\"Org2\":0}"
+		Then The gateway named mycouchgateway has a event type response containing "status":"VALID"
 		When I modify mycouchgateway to submit a transaction with args [createCar,1004,Ford,Mustang,Silver,Andy] for contract fabcar instantiated on channel gatewaychannel using handler option NETWORK_SCOPE_ALLFORTX
-		Then The gateway named mycouchgateway has a event type response matching "{\"Org1\":1,\"Org2\":0}"
+		Then The gateway named mycouchgateway has a event type response containing "status":"VALID"
 		When I modify mycouchgateway to submit a transaction with args [createCar,1005,Ford,Mustang,Silver,Andy] for contract fabcar instantiated on channel gatewaychannel using handler option NETWORK_SCOPE_ANYFORTX
-		Then The gateway named mycouchgateway has a event type response matching "{\"Org1\":1,\"Org2\":0}"
-
-	Scenario: Using a Gateway to evaluate transactions I can use different query handler strategies
```

Comment:
No scenario tests should be removed otherwise we are not regression testing the refactor